superset-frontend/cypress-base/cypress/integration/dashboard/nativeFilters.test.ts

```diff
@@ -61,9 +61,12 @@ describe('Nativefilters', () => {
       .click()
       .type('Country name');
 
-    cy.get('.ant-modal').find('[data-test="datasource-input"]').click();
+    cy.get('.ant-modal')
+      .find('[data-test="datasource-input"]')
+      .click()
+      .type('wb_health_population');
 
-    cy.get('[data-test="datasource-input"]')
+    cy.get('.ant-modal [data-test="datasource-input"] .Select__menu')
```

Comment:
Not a problem, but I'm curious why we need the `.ant-modal` selector here. Mainly wondering if our data-test attribute is too general and should just be made more specific.

Reply:
Oh, I think that's a copy-paste artifact